dbt_subprojects/tokens/models/transfers_and_balances/polygon/tokens_polygon_base_transfers.sql

```diff
@@ -15,7 +15,8 @@
     traces = source('polygon','traces'),
     transactions = source('polygon','transactions'),
     erc20_transfers = source('erc20_polygon','evt_transfer'),
-    native_contract_address = null
+    native_contract_address = null,
```

Comment:
```suggestion
    native_contract_address = '0x0000000000000000000000000000000000001010',
```

Reply:
in order to properly assign `token_standard` column, we need to supply native address here to trigger the case statement:
https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/tokens/macros/transfers/transfers_base.sql#L41-L50